docs/apis/mos/raw_data_pull_api_tokenv2.yaml

```diff
@@ -1044,6 +1064,30 @@ components:
       schema:
         type: string
         format: date
+    Agency:
+      in: query
+      name: agency
+      description: |
+        Use to filter by a specific agency
+
+        Note: that if you use both the agency filter and media source filters, the media source filters don’t apply for non-transparent agencies. This means that for non-transparent agencies, all data is included; not just data from the filtered media sources
```

Comment:
There is a text mistake here. Should be:

Note: If you use both the agency filter and media source filters, the media source filters don’t apply for non-transparent agencies. This means that for non-transparent agencies, all data is included, not just the data from the filtered media sources

Reply:
Fixed the description